Makefile

```diff
@@ -523,6 +529,8 @@ PREVIOUS_TAG ?= $(shell git tag -l | grep -E "^v[0-9]+\.[0-9]+\.[0-9]+$$" | sort
 RELEASE_ALIAS_TAG := $(PULL_BASE_REF)
 RELEASE_DIR := out
 RELEASE_NOTES_DIR := _releasenotes
+USER_FORK ?= $(shell git config --get remote.origin.url | cut -d/ -f4)
+IMAGE_REVIEWERS ?= $(shell ./hack/get-project-maintainers.sh)
```

Comment:
An alternative to parsing `OWNER_ALIASES` is to directly tag `@kubernetes-sigs/cluster-api-maintainers`. This way you won't need the script to parse maintainers. It is similar to the default set in `kpromo`.

Reply:
it doesn't look like `/assign` works for github teams https://prow.k8s.io/command-help#assign

https://github.com/kubernetes-sigs/cluster-api/pull/5713#issuecomment-976972714

Reply:
TIL. This is odd. :(